guides/v2.3/frontend-dev-guide/themes/theme-create.md

```diff
@@ -191,6 +191,18 @@ It is likely that your theme will also contain module-specific files, which are
 {: .bs-callout .bs-callout-info }
 When you change files stored here during theme development, you need to clear the `pub/static` and `var/view_preprocessed` directories, and then reload the pages. Otherwise, the old versions of files are displayed on the storefront.
 
+Run the command below to clear the `pub/static` directory:
+
+```bash
+rm -rf <magento_root>/pub/static/*
+```
+
+and the next one, to clear the `var/view_preprocessed` directory:
```

Comment:
```suggestion
#### To clear the `var/view_preprocessed` directory:
```

Reply:
**implemented**

@hguthrie, all requested changes have been implemented. Please review.

Thank you!